plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2064,271 @@ def coupling_parameter(
+        >>> T_b = 1e3 * u.eV
+        >>> Coulomb_log = 10 * u.dimensionless_unscaled
+        >>> frequencies_using_temperature = CollisionFrequencies(
+        ...     "e-", "e-", T_a=T_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> frequencies_using_temperature.momentum_loss
+        <Quantity 1.83701432e+11 Hz>
+
+        >>> v_a = 1e5 * u.m / u.s
+        >>> frequencies_using_velocity = CollisionFrequencies(
+        ...     "e-", "e-", v_a=v_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> frequencies_using_velocity.energy_loss
+        <Quantity -9.69828719e+15 Hz>
+
+        See Also
+        --------
+        ~plasmapy.formulary.collisions.Coulomb_logarithm : Evaluates the Coulomb
+            logarithm for two interacting electron species.
+        """
+
+        # Note: This function uses CGS units internally to coincide with our references.
+        # Input is taken in MKS units and then converted as necessary. Output is in MKS units.
+
+        if v_a is None:
+            if T_a is not None:
+                v_a = thermal_speed(T_a, test_particle)
+            else:
+                raise ValueError("Please specify either v_a or T_a.")
+        elif T_a is not None:
+            raise ValueError("Please specify either v_a or T_a, not both.")
+
+        self.test_particle = test_particle
+        self.field_particle = field_particle
+        self.v_a = v_a
+        self.T_a = (
+            T_a
+            if T_a is not None
+            else (0.5 * test_particle.mass * v_a**2 / k_B).to(u.K)
+        )
+        self.n_b = n_b
+        self.T_b = T_b
+        self.Coulomb_log = Coulomb_log
+
+    @cached_property
+    def _mass_ratio(self):
+        return self.test_particle.mass / self.field_particle.mass
+
+    @cached_property
+    def momentum_loss(self):
+        """
+        The momentum loss rate due to collisions.
+        """
+        return (1 + self._mass_ratio) * self.phi * self.Lorentz_collision_frequency
+
+    @cached_property
+    def transverse_diffusion(self):
+        """
+        The rate of transverse diffusion due to collisions.
+        """
+        return (
+            2
+            * ((1 - 1 / (2 * self.x)) * self.phi + self._phi_prime)
+            * self.Lorentz_collision_frequency
+        )
+
+    @cached_property
+    def parallel_diffusion(self):
+        """
+        The rate of parallel diffusion due to collisions.
+        """
+        return (self.phi / self.x) * self.Lorentz_collision_frequency
+
+    @cached_property
+    def energy_loss(self):
+        """
+        The energy loss rate due to collisions.
+        """
+        return (
+            2
+            * (self._mass_ratio * self.phi - self._phi_prime)
+            * self.Lorentz_collision_frequency
+        )
+
+    @cached_property
+    def Lorentz_collision_frequency(self):
+        r"""
+        The Lorentz collision frequency (see Ch. 5 of :cite:t:`chen:2016`) is given
```

Comment:
I think I figured this out - docstrings are supposed to start with a simple 1-sentence 'summary' which I think is what the attribute summary is, which is getting cut off here. If we separate a simple summary out, that should fix it.



```suggestion
        The Lorentz collision frequency.
        
        The Lorentz collision frequency (see Ch. 5 of :cite:t:`chen:2016`) is given
```

Reply:
Ohhh that makes sense. This is why we need to implement #1668 !